docling/backend/msword_backend.py

```diff
@@ -899,23 +917,26 @@ def _handle_text_elements(  # noqa: C901
                     text_tmp = "" if len(split_text_tmp) == 1 else split_text_tmp[1]
 
                     if len(pre_eq_text) > 0:
-                        doc.add_text(
+                        e1 = doc.add_text(
                             label=DocItemLabel.PARAGRAPH,
```

Comment:
Can we use `TEXT` instead of `PARAGHRAPH`. I would like to deprecate `PARAGHRAPH`.